doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -8,226 +8,72 @@ A step by step guide to the GeoServer Compliance Interoperability Test Engine (C
 .. contents::
    :depth: 2
 
+Requirements
+------------
+
+Running the tests requires a linux system with docker and docker-compose installed on it.
+
 Check out CITE tools
 --------------------
 
-The CITE tools are available at https://github.com/jdeolive/geoserver-cite-tools. The README file 
-contains the most update documentation of how to checkout and build the tools. The quick version is::
+The CITE tools are available in the build/cite folder of the GeoServer repository:
 
-  git clone git://github.com/jdeolive/geoserver-cite-tools.git
-  cd geoserver-cite-tools
-  git submodule update --init
-  mvn clean install
+https://github.com/geoserver/geoserver/tree/master/build/cite
   
-Run WFS 1.0 tests
------------------
-
-.. note::
-
-   Running WFS 1.0 tests require PostGIS to be installed on the system.
-
-#. Create a PostGIS user named "cite"::
-
-     createuser cite
-
-#. Create a PostGIS databased named "cite", owned by the "cite" user::
-
-     createdb -T template_postgis -U cite cite
-
-#. Change directory to the ``citewfs-1.0`` data directory and execute the script
-   ``cite_data.sql``::
 
-     psql -U cite cite < cite_data.sql
-
-#. Start GeoServer with the ``citewfs-1.0`` data directory. Example::
-
-     cd <root of geoserver install>
-     export GEOSERVER_DATA_DIR=<root of geoserver sources>/data/citewfs-1.0
-     ./bin/startup.sh
-
-#. Change directory back to the cite tools and run the tests::
-
-     ant wfs-1.0
-     
-   With the following parameters:
-
-   #. ``Capabilities URL``
-      
-        http://localhost:8080/geoserver/wfs?request=getcapabilities&service=wfs&version=1.0.0
-
-   #. ``All`` tests included
-
-      .. image:: tewfs-1_0.jpg
-
-Run WFS 1.1 tests
------------------
+Run the tests
+-------------
 
```

Comment:
I think I need some background here on what is being setup, right now I am learning as I go through each step.

I am happy to draw a picture if that helps makes these docs better :P